lib/omniauth/strategies/artsy.rb

```diff
@@ -12,10 +14,6 @@ def self.configure
 
       configure
 
-      def request_phase
-        super
-      end
```

Comment:
Intentional removal?

Reply:
Yeah, this was a RuboCop violation that I let it fix:

```
lib/omniauth/strategies/artsy.rb:17:7: W: Lint/UselessMethodDefinition: Useless method definition detected.
      def request_phase ...
      ^^^^^^^^^^^^^^^^^

10 files inspected, 1 offense detected, 1 offense auto-correctable
```

https://docs.rubocop.org/rubocop/cops_lint.html#lintuselessmethoddefinition